### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by bmicklea**
* Revamped README
* Added links to CONTRIBUTING
* Rewrote Why KitOps
* Tweaked Overview

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,31 @@
...
+KitOps simplifies the handoffs between data scientists, application developers, and SREs working with LLMs and other AI/ML models. KitOps' ModelKits are a standards-based package for models, their dependencies, configurations, and codebases. ModelKits are portable, reproducible, and work with the tools you already use.
+
+### Features
+
+* 🎁 **[Unified packaging](https://kitops.ml/docs/modelkit/intro.html):** A ModelKit package includes models, datasets, configurations, and code. Add as much or as little as your project needs.
+* 🏭 **[Versioning](https://kitops.ml/docs/cli/cli-reference.html#kit-tag):** Each ModelKit is tagged so everyone knows which dataset and model work together.
+* 🤖 **[Automation]](https://github.com/marketplace/actions/setup-kit-cli):** Pack or unpack a ModelKit locally or as part of your CI/CD workflow for testing, integration, or deployment.
+* 🔒 **[Tamper-proofing](https://kitops.ml/docs/modelkit/spec.html):** Each ModelKit package includes a SHA digest for itself, and every artifact it holds.
+* ✅ **Self-verifying:** ModelKits are checked for validity and tampering before and after every action.
```
Comment (gorkem) [inline README.md:28]: Is this is referring to something different than the SHA digest which is already covered on the above line? If not we should remove it.

**Review by gorkem — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,17 +20,17 @@ If anything doesn't make sense, or doesn't work when you try it, please open a b
...
+* [Builds and CI/CD](https://github.com/jozu-ai/kitops/issues?q=is%3Aopen+is%3Aissue+label%3Abuild)
+* Release management
 * Issue triage
 * Answering questions on Discord, or the mailing list
 * Web design
 * Communications, social media, blog posts, or other marketing
-* Release management
 
-Not everything happens through a GitHub pull request. Please contact us in the [#general channel of our Discord server](https://discord.gg/3eDb4yAN) or during our [office hours meeting](./GOVERNANCE.md#meetings) and let's discuss how we can work together.
+If you think there's something else you can help with please contact us in the [#general channel of our Discord server](https://discord.gg/3eDb4yAN) or during our [office hours meeting](./GOVERNANCE.md#meetings) and let's discuss how we can work together.
```
Comment (javisperez) [inline CONTRIBUTING.md:33]:
```suggestion
If you think there's something else you can help with please contact us in the [#general channel of our Discord server](https://discord.gg/Tapeh8agYy) or during our [office hours meeting](./GOVERNANCE.md#meetings) and let's discuss how we can work together.
```

Comment (javisperez) [reply CONTRIBUTING.md:33]: wrong discord link

File: README.md (hunk 1 above)
Comment (bmicklea) [reply README.md:28]: Moving conversation to Discord

Comment (bmicklea) [reply README.md:28]: Agreed to rewrite the Self-verifying section to focus on signing and make it a coming soon feature.

**Review by gorkem — APPROVED** [review]

**Pull request merged**